TestAssets/TestProjects/TestMscorlibReference/project.json

```diff
@@ -0,0 +1,16 @@
+{
+  "version": "1.0.0-*",
+  "dependencies": {},
+  "frameworks": {
+    "netstandardapp1.5": {
+      "dependencies": {
+        "NETStandard.Library": "1.5.0-rc2-23931"
```

Comment:
Actually I know as part of https://github.com/dotnet/cli/pull/2331 @eerhardt is also removing netstandardapp references and instead switching to netcoreapp1.0 which might be necessary here as well.